README.md

```diff
@@ -1,2 +1,34 @@
 # Reviews
-This is the "Reviews" module in our Staybnb web application.
+This is the "Reviews" module in our TopBunk web application.
+
+
+## Related Projects
+
+  - https://github.com/TopBunkNYC/Description
+  - https://github.com/TopBunkNYC/Booking
+  - https://github.com/TopBunkNYC/Neighborhood 
+
+## Development
+To run the application in non-development mode from within the root directory: 
+```sh
```

Comment:
What is ```sh ``` ?

Reply:
with backticks

Reply:
For Markdown syntax, the three backticks signify the beginning of a code snippet, and if you insert the corresponding language abbreviation Github will perform syntax highlighting appropriate for that language. `sh` triggers `bash` syntax highlighting (and `js` would trigger JavaScript syntax highlighting).